sup3r/preprocessing/data_handling/dual_data_handling.py

```diff
@@ -162,24 +162,27 @@ def normalize(self, means, stdevs):
             dimensions (features)
             array of means for all features with same ordering as data features
         """
-        logger.info('Normalizing low resolution data.')
+        logger.info('Normalizing low resolution data features='
+                    f'{self.features}')
         self._normalize(data=self.lr_data,
                         val_data=self.lr_val_data,
                         means=means,
                         stds=stdevs,
                         max_workers=self.lr_dh.norm_workers)
-        logger.info('Normalizing high resolution data.')
+        logger.info('Normalizing high resolution data features='
+                    f'{self.output_features}')
+        indices = [self.features.index(f) for f in self.output_features]
         self._normalize(data=self.hr_data,
                         val_data=self.hr_val_data,
-                        means=means,
-                        stds=stdevs,
+                        means=means[indices],
+                        stds=stdevs[indices],
                         max_workers=self.hr_dh.norm_workers)
 
     @property
     def output_features(self):
         """Get list of output features. e.g. those that are returned by a
         GAN"""
-        return self.hr_dh.output_features
+        return self.lr_dh.output_features
```

Comment:
This is surprising, maybe i'm missing something, why wouldnt the high-res data handler have the correct output feature list? Just because the low-res handler has a list of "train only" features that makes this correct?